res/Sectors/M1105/Folgore.txt

```diff
@@ -1,389 +1,389 @@
-0909 Nokune               C555A87-7 Hi                           {  0 } (B97-3) [EA47] - M  - 514  8 HvFd M0 V
-1005                      A560687-A De Ni Ri                     {  3 } (A59+5) [598C] - KM - 904 12 HvFd G6 IV
-1007                      A878585-D Ag Ni                        {  2 } (E44-3) [673B] - K  - 302 15 HvFd M9 V
-1008                      A686852-A Ri Ga Ph Pa                  {  3 } (97B-1) [7B6D] - K  - 514 16 HvFd G6 V
-1102                      C547226-5 Lo GurvW Pz                  { -2 } (711+0) [6186] - M  A 802  6 HvFd G6 V
-1104                      A436513-E Ni                           {  1 } (E45+0) [561E] - -  - 813 13 HvFd G9 V M0 V
-1110                      D624411-7 Ni                           { -3 } (531-1) [311A] - -  - 704 12 HvFd G6 V M2 V
-1201                      C756514-9 Ag Ni Ga                     {  0 } (C43+0) [7576] - M  - 724 19 HvFd M4 V M7 V 
-1203                      E511000-0 Ba Ic                        { -3 } (300+1) [0000] - -  - 024 14 HvFd M0 V M3 V
-1205 Eloqim               A457987-D Cp Hi                        {  3 } (78C+3) [BCAF] - E  - 914 10 HvFd K6 V M3 V
-1209                      E468510-4 Ag Ni Pr                     { -2 } (840-1) [8328] - -  - 820 12 HvFd M4 V
-1210                      C678410-6 Ni Pa                        { -2 } (C30+1) [1217] - -  - 103 13 HvFd G7 V
-1304                      E300000-0 Ba Va                        { -3 } (700-3) [0000] - -  - 022 10 HvFd G2 V M8 V
-1305                      D564766-6 Ag Ri O:1104                 {  0 } (966-1) [8771] - -  - 700  5 HvFd F7 V
-1306                      C332220-7 Lo Po                        { -2 } (811-3) [2129] - M  - 601 11 HvFd M7 V M8 V
-1307                      C160684-A De Ni Ri                     {  1 } (553+5) [678B] - M  - 603  8 HvFd K6 V
-1309                      C773726-7 Pi                           { -1 } (267-1) [5626] - -  - 704 15 HvFd G3 V M4 V 
-1310                      E797520-5 Ag Ni                        { -2 } (643+0) [5312] - -  - 804 14 HvFd M5 V
-1402                      A525756-C Pi GurvW Da                  {  2 } (C68+1) [695E] - E  A 803 15 HvFd K2 V
-1404 Iilregon             B966A85-C Hi                           {  3 } (995+1) [8D5E] - K  - 712  7 HvFd M4 V M7 V 
-1406                      E64A000-0 Ba Wa                        { -3 } (300+0) [0000] - -  - 010  9 HvFd M5 V
-1410                      D412526-7 Ic Ni Gurv8                  { -3 } (741+4) [2268] - -  - 114 10 HvFd M1 V
-1502                      B624486-7 Ni GurvW Pz                  { -1 } (332+3) [138B] - K  A 715 10 HvFd M6 IV M5 V
-1505                      B400126-C Lo Va GurvW Pz               {  1 } (801+2) [1229] - K  A 500  3 HvFd G3 V
-1506                      D536126-6 Lo GurvW Pz                  { -3 } (801+3) [2169] - -  A 805 16 HvFd M7 V
+0102 Kooask               A79A220-E Lo Wa                        {  1 } (B11+1) [332F] - E  - 523 13 HvGr K0 V
```

Comment:
`HvGr` is not a documented T5SS 4-character allegiance code.

https://travellermap.com/doc/secondsurvey#allegiance

Use 2-character codes for custom allegiances.